modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqexecEngine.scala

```diff
@@ -379,5 +380,6 @@ object SeqexecEngine {
 
     }
   }
+  // scalastyle:on
```

Comment:
This is a large block where scalastyle is off. I'm assume there are several warnings but can't see what could it be.